test/e2e/README.md

```diff
@@ -192,6 +192,39 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on existing K8s cluster
+
+If you already have a K8s cluster on which antrea has been pushed, the following steps should be done for running the e2e tests.
+
+First, you should create the `config` file under the default ssh directory and add the ssh information of each node in the cluster to it. You can add the ssh information like the example below:
```

Comment:
I think there is an option to allow user to pass the path of ssh, please also add it here or below in command line --remote.sshconfig=$path, so user can learn that no need to be default ssh path, if they place file in other directory, please help to verify it. thanks.

Reply:
OK，I will have a try.